sunpy/coordinates/tests/test_spice.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+import spiceypy
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord, frame_transform_graph
+from astropy.tests.helper import assert_quantity_allclose
+from astropy.utils.data import download_file
+
+from sunpy.coordinates import spice
+from sunpy.time import parse_time
+
+
+@pytest.fixture(scope="module", params=[pytest.param(None, marks=pytest.mark.remote_data)])
+def spice_test():
+    # We use the cache for the SPK file because it's also used in other tests (does cache work?)
+    spk = download_file("https://naif.jpl.nasa.gov/pub/naif/generic_kernels/spk/planets/de440s.bsp",
+                        cache=True)
+
+    # Leapseconds (LSK) and physical constants (PCK)
+    lsk = download_file("https://naif.jpl.nasa.gov/pub/naif/generic_kernels/lsk/naif0012.tls")
+    pck = download_file("https://naif.jpl.nasa.gov/pub/naif/generic_kernels/pck/pck00011_n0066.tpc")
+
+    # Use frames (FK) from SunSPICE
+    fk = download_file("https://soho.nascom.nasa.gov/solarsoft/packages/sunspice/data/heliospheric.tf")
+
+    spice.initialize([spk, lsk, pck, fk])
+
+    yield
+
+    # The fact that we need to do this suggests that the cache is not persistent across tests...
```

Comment:
I am not sure I follow you here, what cache?

Reply:
My hope was that by using Astropy`s `download_file()`, I would be using the same cache that is used when downloading the DE440s JPL ephemeris in other unit tests.  However, I suspect that `pytest` is doing too good of a job of isolation (without setting things to session scope).